### DISCUSSION TITLE
Benutzer Rechte Ordner

### DISCUSSION THREAD

**Discussion by danny1605** [grafana/grafana#73308] [General]
Hallo kann mir bitte jemand erklären wie ich einzelne Benutzer rechte auf Ordner gebe ? Die User die derzeit angelegt sind haben alle view rechte und können in allen Ordner rein. Wie lege ich fest das User xy nur Zugang zu Ordner xy hat ?

**Comment by gelicia** [comment]
https://grafana.com/docs/grafana/latest/administration/user-management/manage-dashboard-permissions/
